typescript/cli/src/check/warp.ts

```diff
@@ -38,3 +48,40 @@ export async function runWarpRouteCheck({
 
   logGreen(`No violations found`);
 }
+
+function formatCheckerInput({
+  warpRouteConfig,
+  onChainWarpConfig,
+}: {
+  warpRouteConfig: Readonly<HypTokenRouterConfig>;
+  onChainWarpConfig: HypTokenRouterConfig;
+}) {
+  // If the hook config is not defined in the input file,
+  // we need to remove it from the onChainWarpConfig if it was derived
+  if (!warpRouteConfig.hook) {
+    onChainWarpConfig.hook = undefined;
+  }
+
+  // if the hook config is defined the input file, it means the user wants to check
+  // the hook config, so we need to add the default hook address to the onChainWarpConfig
+  // in case the default hook is currently used by the token.
+  if (warpRouteConfig.hook && !onChainWarpConfig.hook) {
+    onChainWarpConfig.hook = constants.AddressZero;
+  }
+
+  // If the ism config is not defined in the input file,
+  // we need to remove it from the onChainWarpConfig if it was derived
```

Comment:
Why do we remove it? it the ISM is supposed to be the default, but the actualy ISM derived is something else, sholdn't we show that?